config/default-config.yml

```diff
@@ -346,6 +353,8 @@ network-config:
             # opportunistically graft peers that are not access nodes (i.e., with -1),
             # or penalized peers (i.e., with -100).
             opportunistic-graft: 101
+            # The threshold at which the duplicate message count for a peer will result in the peer being penalized
+            duplicate-message: 10e+4
```

Comment:
Please note that this parameter is not part of the `internal` category. The `internal` category includes parameters injected directly into the libp2p pubsub system during pubsub router construction. These parameters govern the GossipSub peer scoring's internal logic, which is external to the Flow code. The peer scoring logic defined in Flow is under the `application-specific` category, where you've placed `duplicate-message-penalty`. My suggestion is to rename `duplicate-message` to `duplicate-message-threshold` and position it just before `duplicate-message-penalty` as part of the node's `application-specific` score.

Reply:
https://github.com/onflow/flow-go/pull/5058/commits/5d20d5e7fac783fdae8ad13c819cd9f0f5b98988